### PR TITLE
chore(IT Wallet): [SIW-1745] IT Wallet copy review

### PR DESCRIPTION
**Pull request by ale-mazz**
This pull request includes various updates to the localization files `locales/en/index.yml` and `locales/it/index.yml` to improve the consistency and clarity of the translated text. The most important changes include updates to terminology, content adjustments for better readability, and corrections to capitalization.

## List of changes proposed in this pull request
- Various modifications to the IT Wallet copy

## How to test
- Test a complete ITW flow, starting with the eID obtainment, then adding a credential, checking every banner/bottom sheet, removing the credential, and revoking the eID.

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -3271,16 +3271,16 @@ features:
...
-      tos: Per maggiori informazioni, leggi l’[informativa Privacy](https://io.italia.it/informativa-sperimentazione) e i [Termini e Condizioni d’uso](https://io.italia.it/tos-sperimentazione)
+      content: "###### Documenti su IO: come funziona \n Da oggi puoi aggiungere al Portafoglio di IO **la versione digitale dei tuoi documenti** personali, come la Patente di guida e la Tessera Sanitaria.  \n\n Attiva la funzionalità **Documenti su IO** per averli sempre a portata di mano sul tuo dispositivo.  \n ###### È facile e veloce \n Avrai bisogno delle credenziali **SPID** o **CIE** (Carta d’Identità Elettronica) per completare l’attivazione: è un passaggio di sicurezza necessario per garantire la sicurezza dei tuoi dati."
+      tos: Premendo **Continua** dichiari di aver letto l’[informativa Privacy](https://io.italia.it/informativa-sperimentazione) e i [Termini e Condizioni d’uso](https://io.italia.it/tos-sperimentazione).
```
Comment (LazyAfternoons) [inline locales/en/index.yml:3275]: Can we use the one stored in the `.env` file and use i18n concatenation to handle this URL?

**Review by LazyAfternoons — CHANGES_REQUESTED** [review]

File: locales/en/index.yml [hunk 2]
```diff
@@ -3315,11 +3315,11 @@ features:
...
       credentialAuth:
         title: "{{credentialName}}: dati necessari"
-        subtitle: "Saranno condivisi con **{{organization}}** per il rilascio della credenziale"
+        subtitle: "Saranno condivisi con **{{organization}}** per il rilascio della versione digitale del documento."
         requiredClaims: Dati richiesti
         disclaimer:
           0: I tuoi dati sono al sicuro e saranno trattati solo per le finalità descritte in informativa Privacy.
-          1: I dati saranno condivisi solo per il tempo necessario al rilascio della credenziale.
+          1: I dati saranno condivisi solo per il tempo necessario al rilascio della versione digitale del documento.
         tos: "Premendo **Continua** dichiari di aver letto e compreso l’[Informativa Privacy](https://io.italia.it/informativa-ipzs-sperimentazione/)."
```
Comment (gispada) [inline locales/en/index.yml:3323]: Is this privacy url (https://io.italia.it/informativa-ipzs-sperimentazione) valid? I get 404.

Comment (hevelius) [reply locales/en/index.yml:3323]: the URL in env file is valid for me https://io.italia.it/informativa-sperimentazione

Comment (ale-mazz) [reply locales/en/index.yml:3323]: The correct one is https://io.italia.it/informativa-ipzs Thanks for the catch!

File: locales/it/index.yml [hunk 3]
```diff
@@ -3271,16 +3271,16 @@ features:
...
-      tos: Per maggiori informazioni, leggi l’[informativa Privacy](https://io.italia.it/informativa-sperimentazione) e i [Termini e Condizioni d’uso](https://io.italia.it/tos-sperimentazione)
+      content: "###### Documenti su IO: come funziona \n Da oggi puoi aggiungere al Portafoglio di IO **la versione digitale dei tuoi documenti** personali, come la Patente di guida e la Tessera Sanitaria.  \n\n Attiva la funzionalità **Documenti su IO** per averli sempre a portata di mano sul tuo dispositivo.  \n ###### È facile e veloce \n Avrai bisogno delle credenziali **SPID** o **CIE** (Carta d’Identità Elettronica) per completare l’attivazione: è un passaggio di sicurezza necessario per garantire la sicurezza dei tuoi dati."
+      tos: Premendo **Continua** dichiari di aver letto l’[informativa Privacy]({{privacyUrl}}) e i [Termini e Condizioni d’uso](https://io.italia.it/tos-sperimentazione).
```
Comment (gispada) [inline locales/it/index.yml:3275]: Did you forget to use a parameter for the TOS url?

Comment (ale-mazz) [reply locales/it/index.yml]: Thanks! Fixed.

**Review by gispada — APPROVED** [review]
LGTM

**Review by LazyAfternoons — APPROVED** [review]

**Pull request merged**